WordPressKit/WordPressComRestApi.swift

```diff
@@ -498,10 +500,23 @@ open class WordPressComRestApi: NSObject {
             request.httpBody = parameters.percentEncoded()
         }
 
-        return try await urlSession.data(for: request, delegate: nil)
+        return try await withCheckedThrowingContinuation { continuation in
```

Comment:
`urlSession.data(for:)` is only available in iOS15, so I had to wrap the method in async/await myself

Reply:
[Jetpack/WordPress will soon support iOS 15 and above only](https://github.com/wordpress-mobile/WordPress-iOS/pull/20917) and [WooCommerce already does](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/WooCommerce.xcodeproj/project.pbxproj#L13591).

Your diff doesn't add much complexity, but I wonder if it would be cleaner to bump the library to iOS 15 instead.